misc/testerina/modules/testerina-core/src/main/ballerina/src/test/assert.bal

```diff
@@ -67,14 +67,9 @@ public function assertFalse(boolean condition, public string msg = "Assertion Fa
 # + actual - Actual value
 # + expected - Expected value
 # + msg - Assertion error message
-public function assertEquals(any|error actual, any|error expected, public string msg = "Assertion Failed!") {
+public function assertEquals(anydata|error actual, anydata|error expected, public string msg = "Assertion Failed!") {
     boolean isEqual = false;
```

Comment:
Is the assignment required? The very next line is `isEqual = (actual == expected)`. 
Im assuming changing it to `boolean isEqual = (actual == expected)` should work. 

Same for line 87